.github/workflows/pr-e2e.yml

```diff
@@ -130,8 +130,9 @@ jobs:
         continue-on-error: true
         id: test
         env:
-          AWS_RUN_IDENTITY_TESTS: true
+          AWS_RUN_IDENTITY_TESTS: true          
```

Comment:
could you please fix the whitespace here^

Reply:
yes, nice catch!